app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(command: str) -> None:
+    names = command.split()
```

Comment:
```suggestion
    command, source, destination = command.split()
```
it's more clear to use unpacking here